### ISSUE TITLE
Harpoon window won't close

### ISSUE THREAD

**Issue by alesbrelih** [ThePrimeagen/harpoon#201]
Hi!

I was wondering if you could help me solve a mystery (dramatic sound on)..

I've tried reinstalling the plugin after a while and I've noticed a bug which I can't quite figure it out. Basically everything works but I can't really close the Harpoon window.

First screen shows what happens after I've added a mark to the harpoon which is 👌 
<img width="1055" alt="Screenshot 2022-09-01 at 23 58 08" src="https://user-images.githubusercontent.com/10506436/188020309-998c2455-084a-4e09-8f8b-b8721e8eaecb.png">

This second screen is what I see after I "close" harpoon either with 'q', ':wq' or any other thing.

<img width="1043" alt="Screenshot 2022-09-01 at 23 58 17" src="https://user-images.githubusercontent.com/10506436/188020399-d7228217-a942-4bd7-bab6-30fc0c71cad1.png">

Basically content of Harpoon gets hidden but the window is still there.
I tried to pinpoint the problem somewhere and I found a solution which for which I don't know why it works.

Basically if I remove https://github.com/ThePrimeagen/harpoon/blob/master/lua/harpoon/ui.lua#L98 everything starts working 🤔 
Any ideas what is going on?

I'm using neovim nightly and running it on mac 🤔 

Thanks!
